src/node_crypto.cc

```diff
@@ -211,6 +211,95 @@ static int NoPasswordCallback(char* buf, int size, int rwflag, void* u) {
 }
 
 
+namespace error {
```

Comment:
maybe this name could be more specific since it relates to OpenSSL and not the general node error mechanism?

Reply:
This is `node::crypto::error`, though that's not obvious from the context diff. I was going to move soe more of the crypto error stuff into this namespace, but not in this PR, its not related. Or I could rename the functions to `DecorateError()`, as you all prefer.